src/components/atoms/Toggler/Toggler.tsx

```diff
@@ -1,28 +1,38 @@
 import React from "react";
+import classNames from "classnames";
 
-import { Checkbox, CheckboxProps } from "components/atoms/Checkbox";
+import "./Toggler.scss";
 
-export interface TogglerProps
-  extends Omit<CheckboxProps, "toggler" | "checked" | "defaultChecked"> {
+export interface TogglerProps {
   toggled?: boolean;
   title?: string;
-  defaultToggled?: boolean;
+  onChange?: () => void;
+  containerClassName?: string;
+  name?: string;
```

Comment:
I'm thinking maybe some of  these changes are just because it's a fork from an older branch, or maybe leftovers from branch switching.

Reply:
moved toggler to `attendee` components + updated styles [8bf8d88](https://github.com/sparkletown/sparkle/pull/2840/commits/8bf8d88aa41daa2e728044cf16a1ace637c3f9db)